src/data-tutorials/users.tsx

```diff
@@ -831,6 +831,13 @@ const Users: User[] = [
     website: 'blog/tutorials/flutter-celo-easily-build-flutter-mobile-dApps',
     tags: ['intermediate','foundation', 'flutter', 'composer'],
   },
+  {
+    title: 'How to build a Basic CRUD App in Solidity',
+    description: 'This post will teach you how to create a basic Create, Read, Update and Delete (CRUD) smart contract and test it with the Truffle framework.',
+    preview: require('./showcase/celo-sage.png'),
```

Comment:
For the showcase image you can get it on your trello card, save it in the `src/data-tutorials/showcase/{beginner/intermediate/advandced} folder